src/haddock/modules/topology/topoaa/cns/generate-topology.cns

```diff
@@ -155,7 +142,8 @@ define(
 
 {* nucleic acid linkage file *}
 {* use TOPPAR:toppar/dna-rna-pho.link for 5'-phosphate *}
-{===>} nucl_link_infile="TOPPAR:dna-rna-1.3.link";
+{* Commented out because provided by input parameter $nucl_link_infile *}
+!{===>} nucl_link_infile="TOPPAR:dna-rna-1.3.link";
```

Comment:
I would remove completely these five lines - then we no longer have any link-related lines in the definition section of the CNS script